doc/_templates/navbar.html

```diff
@@ -8,7 +8,8 @@
   </button>
   <ul class="dropdown-menu" aria-labelledby="dropdownMenu1">
     <li><a href="https://mne-tools.github.io/dev/index.html">Development</a></li>
-    <li><a href="https://mne-tools.github.io/stable/index.html">v0.18 (stable)</a></li>
+    <li><a href="https://mne-tools.github.io/stable/index.html">v0.19 (stable)</a></li>
+    <li><a href="https://mne-tools.github.io/stable/index.html">v0.18</a></li>
```

Comment:
This should had been 0.18 not stable but still does not explain why stavel is not 0.19